doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -370,9 +372,8 @@ specified by the **geometry** parameter:
   is the center of the circle.
 + In the case of `polygon` includes a list of elements, which are the vertices of the polygon.
 
-# REST API
 
-# Group NGSIv2
+# Group Root
```

Comment:
I think this breaks the render:

Current:

![righ-render](https://cloud.githubusercontent.com/assets/1534240/10136367/61f17870-65f4-11e5-8725-2719c6753427.png)

PR:

![bad-render](https://cloud.githubusercontent.com/assets/1534240/10136354/59254eec-65f4-11e5-885d-cb1fb9bac7b6.png)